app/assets/javascripts/components/Newsletter.js

```diff
@@ -0,0 +1,72 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+
+  'use strict';
+
+  var Newsletter,
+      defaultConfig = {};
+
+  Newsletter = function($el, config) {
+    Newsletter.baseConstructor.call(this, $el, config, defaultConfig);
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(Newsletter);
+
+  /**
+  * Set up and populate the model from the form inputs
+  * @param {Promise} initialised
+  */
+  Newsletter.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this.showInArticle($);
+  };
+
+  Newsletter.prototype.showInArticle = function($) {
+    var $panel = this.getPanel($);
+    var $article = $('.editorial main');
```

Comment:
Should/can this be namespaced at all? Or targetted in a more generic way via a data attribute?